### DISCUSSION TITLE
custom pre optimization predicted target

### DISCUSSION THREAD

**Discussion by ohad-monday** [google/lightweight_mmm#125] [Ideas]
Hey
It's sometime more useful to have the pre vs post budget allocation on a more relevant in time period (last vs next 90 days for example) - pre period could be somehow customed?

calculating the budget allocation of the last x days is easy off course.
But how is it possible to calculate the pre optimization predicted target based on a previous budget (according to the last 30/90 days of budget allocation)?
![image](https://user-images.githubusercontent.com/61185809/209085876-07c4b86e-54b6-4267-a56f-0023327d8773.png)

**Comment by michevan** [comment]
Thanks a lot for flagging this! I just wanted to share a link to our new "[support](https://github.com/google/lightweight_mmm#support)" section in the readme to give you some more information on how we can and cannot support such ideas at the moment.
